### PR TITLE
feat: added Text utilities

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:

#Description: Added Text Utilties Component
1) Headings
2) Paragraphs
3) Aligned text
4) Text wrapping and overflow
5) Text transform
6) Font weight and italics
7) Text decoration

Screenshots:
heading
para
other-comp

### CODE REVIEW

File: Components/TextUtilities/textutilities.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+          <div class="txt-container space-evenly wrap">
+            <p class="text-upperCase">Uppercased text</p>
+            <p class="text-lowerCase">Lowercased text</p>
+            <p class="text-capitalCase">Capitalized ext</p>
+          </div>
+
+          <div class="baseline__frame center">
+            <iframe
+              class="iFrame"
+              class="iFrame"
```
Comment (teja538) [inline Components/TextUtilities/textutilities.html]: remove one class attribute

**Review by teja538 — COMMENTED** [review]
naming is good and its easily readable
a small suggestion : I think using margin: 2rem for text-fs1,fs2 .....in utility.css may effect further usage when we use the component library for other projects. you can define a separate class for giving margin to that block. If I am wrong please let me know

**Review by Asuraking1n — APPROVED** [review]
Everything looks cool....you can merge

File: Components/TextUtilities/textutilities.html (hunk 1 above)
Comment (krgarima) [reply Components/TextUtilities/textutilities.html]: done

**Pull request merged**